app/components/workspace/json-templates/admin-workspace.json

```diff
@@ -0,0 +1,16 @@
+{
+    "metadata": {
```

Comment:
why not move all of those up one level if we have workspace bellow. Also `features` might be better?

Reply:
if you are happy with that then we could do. didn't want metadata mixing with the actual control values.